devito/ir/iet/nodes.py

```diff
@@ -834,6 +886,49 @@ def is_tensor(self):
         return False
 
 
+class Lambda(Node):
```

Comment:
I don't have a strong opinion, but could be called `LambdaC` (or something along those lines) to make it a little clearer? That's probably making it a little less 'nice' though.

Reply:
it'd actually need to be LambaCPP (C doesn't have lambdas), so I guess I'll just stick to Lambda, it don't think it's a potential source of confusion after all